Directory.Build.props

```diff
@@ -1,4 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
-  <Import Project="scripts\build\TestFx.props" />
-</Project>
+  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
+  <!-- <Import Project="eng/Versions.props" /> -->
```

Comment:
Is it temporary? If so could we add a comment about why, otherwise it feels like dead code and should be removed.

Reply:
It will be removed - now I use to make sure when build by arcade fails it is on us and not arcade. It won't be in final code.